source/iface/queue.rst

```diff
@@ -116,23 +128,30 @@ invalid_object_error
 
   backend get_backend() const noexcept;
 
+Returns a backend identifying the SYCL ``backend``
+associated with this ``queue``.
+
 ``get_context``
 ===============
 
 ::
 
   context get_context() const;
 
-Returns context associated with queue.
+Returns the SYCL queue’s context. Reports errors using SYCL exception classes.
```

Comment:
This wording about reporting errors seems confusing to me.  I proposed removing it from the spec (KhronosGroup/SYCL-Docs#471).  I suggest removing it from the reference guide too.

Reply:
Fixed with commit: https://github.com/KhronosGroup/SYCL_Reference/pull/74/commits/82e7c5b4823c084ecf102d61b6505693ffc0e63d